common/reducers/transaction/meta/meta.ts

```diff
@@ -14,8 +14,8 @@ import {
 import { Reducer } from 'redux';
 
 const INITIAL_STATE: State = {
-  unit: 'ether',
-  previousUnit: 'ether',
+  unit: 'ETH',
+  previousUnit: 'ETH',
```

Comment:
I'm a little concerned about making this appear correctly, because I wouldn't want us to use `transaction.meta.unit` nakedly anywhere without turning it into network.unit. How would you feel about making this "default" or "network unit" or something like that, to make it clear to anyone that this shouldn't be used for display? That'd also mean we should rename the check function to isNetworkUnit or something like that, for non-Ethereum networks.

Reply:
Yea that's a good idea, I'll make those changes